apps/search/vercel.json

```diff
@@ -0,0 +1,10 @@
+{
+  "functions": {
+    "src/pages/api/webhooks/product_updated.ts": {
+      "memory": 256
+    },
+    "src/pages/api/webhooks/product_variant_updated.ts": {
+      "memory": 256
+    }
```

Comment:
```suggestion
    "src/pages/api/webhooks/saleor/**/*": {
      "memory": 256
    },
```

Reply:
- set the settings for all of the webhooks
- `saleor` directory was missing in the path